serverless/messaging/reference-content/migration.mdx

```diff
@@ -0,0 +1,173 @@
+---
+meta:
+  title: Understanding Messaging and Queuing GA and Migration
+  description: Learn about the new version of Messaging and Queuing being released for when SQS and NATS go into General Availability, and how migration to the new version will work.
+content:
+  h1: Understanding Messaging and Queuing GA and Migration
+  paragraph: Learn about the new version of Messaging and Queuing being released for when SQS and NATS go into General Availability, and how migration to the new version will work.
+tags: messaging queuing sqs sns nats beta ga general-availability migration api v1 v2 namespace no-namespace
+dates:
+  validation: 2023-09-25
+  posted: 2023-09-25
+categories:
+  - serverless
+---
+
+## Overview
+
+Soon, a new version of Messaging and Queuing will be released. In this version, the concept of **namespaces** will be removed, to streamline your workflows, offer you a better developer experience, and simplify your management of your Messaging and Queuing resources.
+
+The release of this new version in the Scaleway console will coincide with the [NATS](/serverless/messaging/concepts/#nats) and [SQS](/serverless/messaging/concepts/#sqs) protocols leaving [Public Beta](https://www.scaleway.com/en/betas/) and going into General Availability (GA) This means that the Messaging and Queuing product, when used with these protocols, will become billable. [SNS](/serverless/messaging/concepts/#sns) will remain in Public Beta for now while we continue testing and improving the product.
+
+Your NATS resources will automatically be migrated to the new version of the product. However, for SQS and SNS you will need to migrate, by recreating your resources in the new version. You can do this via the API or the console. This is very important, as 30 days after the new version is released, the old version of the product will be decommissioned and all resources still existing within it will be deleted.
```

Comment:
This is very important, as 30 days after the new version is released, the old version of the product will be decommissioned and all resources still existing within it will be deleted.
 
I feel as it's missing a "." somewhere no ?